docs/user-defined-type-functions.md

```diff
@@ -92,10 +92,6 @@ All attributes of newly created typelib are initialized with empty tables / arra
 | `issubtypeof(arg: typelib)` | `boolean` | returns true if self is syntactically a subtype or equal to arg in the type hierarchy |
 | `equalsto(arg: typelib)` | `boolean` | returns true if self is syntactically equal to arg in the type hierarchy |
 
-* `issubtypeof(arg: typelib)` and `equalsto(arg: typelib)` will also overload `__le` and `__eq` respectively
-    * e.g. `t1 <= t2` is equivalent to `t1:issubtypeof(t2)`
-    * e.g. `t1 == t2` is equivalent to `t1:equalsto(t2)`
```

Comment:
I know we said to delete the `<=`, but I thought we compromised by keeping `==` and changing `<=` to `issubtypeof`.